### PR TITLE
Fix Find my school disappearing

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-14663
affects: Parent, Student, Teacher
release note: Fixed a layout issue when logging in with an iPhone SE

### CODE REVIEW

File: Core/Core/Login/LoginStartViewController.storyboard [hunk 1]
```diff
@@ -339,7 +339,7 @@
                                     <constraint firstItem="Vec-s2-2D4" firstAttribute="leading" secondItem="xBc-5O-QU6" secondAttribute="leading" id="GCc-Dr-KkO"/>
                                     <constraint firstAttribute="trailing" secondItem="k72-wA-dRu" secondAttribute="trailing" constant="16" id="NxQ-js-qpq"/>
                                     <constraint firstItem="Op3-kc-wqp" firstAttribute="leading" secondItem="xBc-5O-QU6" secondAttribute="leading" constant="16" id="QqB-yW-Rd2"/>
-                                    <constraint firstAttribute="height" priority="750" constant="225" id="WS4-Gy-0Kt"/>
+                                    <constraint firstAttribute="height" priority="500" constant="225" id="WS4-Gy-0Kt"/>
```
Comment (vanwagonet) [inline Core/Core/Login/LoginStartViewController.storyboard:342]: This was the core fix. Break the height of the previous logins before breaking the height of the login button.

File: Core/Core/Login/LoginStartViewController.swift [hunk 2]
```diff
@@ -371,6 +373,7 @@ extension LoginStartViewController: UITableViewDataSource, UITableViewDelegate,
             view.layoutIfNeeded()
             UIView.animate(withDuration: 0.5) {
                 self.previousLoginsBottom?.constant = -self.previousLoginsView.frame.height
+                self.previousLoginsView.alpha = 0
```
Comment (vanwagonet) [inline Core/Core/Login/LoginStartViewController.swift:376]: Sometimes the view doesn't get all the way down, so this makes it go away, even when the frame.height isn't far enough to move.

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/jbpluw7) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fjbpluw7)
teacher: [Link](https://rebrand.ly/vmbrumj) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fvmbrumj)
parent: [Link](https://rebrand.ly/fyjb5o6) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Ffyjb5o6)

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.16% | 93.16% | 0%

[MBL-14663](https://instructure.atlassian.net/browse/MBL-14663)

#### Affected Apps: Parent, Student, Teacher

#### Release Note: 
Fixed a layout issue when logging in with an iPhone SE
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**